driver-minio/src/main/resources/minio.properties

```diff
@@ -20,3 +20,6 @@ bucketName=sbk
 
 # Recreate the bucket; if already existing , delete it and create it again
 reCreate=false
+
+# No server certificate validation
+insecure=false
```

Comment:
by default , it should be "true"; so that our regular tests works

Reply:
BTW, `-insecure` option has no effect for non-TLS/SSL endpoint. Actually, I'm measuring non TLS/SSL S3 box performance without `-insecure` option (that is equivalent to `-insecure false'). Also, IMHO, if we add TLS/SSL boxes with self-signed certificates for our regular testing, test cases should be rewritten (with `-insecure true').

But, anyway, I will update my PR.